net-libs/liboauth/liboauth-1.0.3-r100.ebuild

```diff
@@ -0,0 +1,87 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="C library implementing the OAuth secure authentication protocol"
+HOMEPAGE="http://liboauth.sourceforge.net/"
+SRC_URI="mirror://sourceforge/project/${PN}/${P}.tar.gz"
+
+LICENSE="|| ( GPL-2 MIT )"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x64-macos"
+IUSE="bindist curl doc libressl +nss ssl"
```

Comment:
Maybe `-ssl`? You specify that `ssl` and `nss` are mutually exclusive in :18, and `nss` is enabled by default. I think in 99% user has a system wide `ssl` flag enabled, and will have to set package specific `-ssl`. Why not to set `-ssl` right away?

Reply:
I think it's better not to enable nss by default and leave it to the user's choice.

Reply:
Maybe. Yet `nss` is enabled by default in the official gentoo ebuild (but there is no `ssl` flag).